agents/trainer/task.py

```diff
@@ -14,19 +14,16 @@
 
 from __future__ import absolute_import, division, print_function
 
-import argparse
 import datetime
 import logging
 import os
 import pprint
 import uuid
 
-import pip
-import tensorflow as tf
 from google.cloud import storage
+import tensorflow as tf
 
 import agents
-import pybullet_envs  # To make AntBulletEnv-v0 available.
```

Comment:
Hey removing 'pybullet_envs' will break the example. By importing it the environments get registered and are then available to the trainer. Just add a "#pylint: disable=unused-import" for that one. Good changes on the rest. I made some of these changes in https://github.com/kubeflow/examples/pull/45/files but didn't deal with the issues cased by that base hparams object. I'm thinking that should just be converted to a dict or an hparams object given it would be pretty unsightly to put an pylint ignore comment on each of those lines.

Reply:
And when I say "base hparams object" I'm referring to hparams_base().